src/main/scala/com/fulcrumgenomics/vcf/filtration/ReadEndSomaticVariantFilter.scala

```diff
@@ -123,27 +124,117 @@ class EndRepairArtifactLikelihoodFilter(val distance: Int = 2,
 
     Map(Info.id -> pMutation)
   }
+}
+
+/** The companion object for [[ReadEndSomaticVariantFilter]]. */
+object ReadEndSomaticVariantFilter {
+
+  /** Returns true if <gt> is a heterozygous SNV with one reference allele, else false */
+  private[filtration] def isPointMismatch(gt: Genotype) : Boolean = {
+    val ref = gt.alleles.ref
+    if (ref.length != 1) false else { // No interest in deletions/MNVs, so only consider if ref length is 1
+      gt
+        .alleles
+        .alts
+        .collect { // Collect all the alt alleles
+          case alt: SimpleAllele // Only interested in simple alleles, no symbolics or other strange stuff
+            if (alt.length == 1 & alt.bases != ref.bases) => alt // No interest in insertions, so only retain alts with length 1
+          }.nonEmpty // If anything remains, at least one of our alleles is a point mismatch
+    }
+  }
```

Comment:
A couple of things:
1. Can we call this `isHetSnv` because `pointMismatch` doesn't actually imply het.
2. This doesn't actually do what you want.  It's only checking the alleles on the VCF record and doesn't inspect the genotype at all.  So it will pass all cases where the REF allele is single-base and _at least one_ single base ALT allele exists.

E.g. it would return `true` for:
```
chr1  1234  . G  A,GAGAGAGA   . PASS  . 1/2
```

It needs to either use the methods on `gt` like `gt.isHet` and/or refer to `gt.calls` which are the called alleles in the genotype.  `gt.alleles` refers to the set of alleles on the enclosing VCF record.

Reply:
@tfenne I share the confusion with @ThomasHSmith between `gt.calls` and `gt.alleles`. I can see in the code that `alleles` is from the `VariantContext.getAlleles()`, and `calls` is from `Genotype.getAlleles()`. I get the utility of having access to the parent Variant's alleles inside the `Genotype`, but from an api stand point I would have expected fgbio's `Genotype.alleles` to relate to htsjdk's `Genotype.getAlleles()` in the same way that `Variant.alleles` relates to `VariantContext.getAlleles()`, with some other member for the `Variant` alleles, like `Genotype.variantAlleles`

Reply:
@mjhipp Feedback on the API in fgbio is welcome.  That said, the HTSJDK API is a horror show and I would generally suggest we not reference that in discussion of the fgbio VCF API!

Reply:
resolved